routes/conversations.js

```diff
@@ -10,6 +10,11 @@ router.put('/:id', (req, res) => {
       return;
     }
 
+    if (conversation.owner.toString() !== req.user.id) {
+      res.status(401).json(err);
```

Comment:
`err` isn't relevant here.  We need to respond with a error describing that the user is unauthorized.